en/simulation/gazebo.md

```diff
@@ -12,31 +16,31 @@ graph LR;
   MAVLink-->SITL;
 {% endmermaid %}
 
-## Installation
 
-The installation requires to install Gazebo and our simulation plugin.
 
-> **Note** Gazebo version 7 is recommended (the minimum version is Gazebo 6). If you run Linux and installed a ROS version earlier than Jade, be sure to uninstall the bundled Gazebo (sudo apt-get remove ros-indigo-gazebo) version as it is too old.
+## Installation
 
-Check the [Linux](../setup/dev_env_linux.md) and [Mac](../setup/dev_env_mac.md) pages for install infos.
+> **Tip** If you plan to use PX4 with ROS you should instead [follow the instructions here](../simulation/ros_interface.md) to install Gazebo as part of ROS! Note that we install Gazebo7 with ROS, as this is the default compatible version.
 
-#### ROS Users
+Gazebo 8 setup in included in our standard build instructions:
+- **macOS:** [Development Environment on Mac](http://localhost:4000/en/setup/dev_env_mac.html)
```

Comment:
@TSC21 Does macOS installer also include ROS Kinetic? If so, does it install Gazebo 7 (the default) or Gazebo 8 (preferred).

Reply:
Note also that I've dropped the mention of gazebo 6. We prefer gazebo 8 as standalone and for ROS we use whatever it default to.

Reply:
I can't actually confirm that since I don't have a Mac...